### PR TITLE
More informative errors on SWIG interface import failures

### PR DESCRIPTION
**Pull request by dweindl**
Related to #954

### CODE REVIEW

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/setuptools.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5016253&bid=16539904)

File: python/amici/petab_import.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def import_petab_problem(
         model_output_dir: str = None,
         model_name: str = None,
         force_compile: bool = False,
-        **kwargs) -> amici.Model:
+        **kwargs) -> 'amici.Model':
```
Comment (FFroehlich) [inline python/amici/petab_import.py]: remove apostrophes?

Comment (dweindl) [reply python/amici/petab_import.py]: They were on purpose. This module is in principle usable without the SWIG interface. If we remove the apostrophes, it is not.

Comment (FFroehlich) [reply python/amici/petab_import.py]: Whats the practical usefulness of typehints without type?

Comment (dweindl) [reply python/amici/petab_import.py]: It's still visible to the implementer. And I thought that most tools would still be able to check.
I still see more value in keeping it like that, then removing it completely or making this module unusable for non-swig installations.

Comment (FFroehlich) [reply python/amici/petab_import.py]: okay :)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/setuptools.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5016253&bid=16539904)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/setuptools.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5016253&bid=16539904)

**Pull request merged**